### DISCUSSION TITLE
Como gerenciar Issues/tasks etc

### DISCUSSION THREAD

**Discussion by sbvorpagel** [frontendbr/forum#772] [Perguntas e Respostas]
Venho estudando e pensando no assunto a algum tempo, como gerenciar Issue, categorizar, verificar Issues duplicadas etc, nos mais diversos tipos de projetos. E nada melhor que recorrer a comunidade.

Vocês utilizam alguma técnica ou ferramenta?
Apenas padrão de abertura de Issue é o suficiente? Ou não é necessário?

**Comment by kvnol** [comment]
Como sempre trampei sozinho (como único dev nas empresas) não sei como funciona isso, por isso estou acompanhando tb.

**Comment by klarkc** [comment]
Cara é muito relativo, técnicas e ferramentas são escolhidas para resolver problemas que você ou sua equipe enfrentam. Claro é importante focar no método ágil de entrega, que tem se mostrado uma das melhores, se não a melhor metodologia de desenvolvimento, mas o próprio método ágil é muito amplo, e pode ser aplicado de muitas formas diferentes. Explica melhor como é seu cenário para a comunidade poder lhe ajudar mais assertivamente.

**Comment by sbvorpagel** [comment]
@klarkc A questão é mais curiosidade mesmo, vejo aqui mesmo nesta comunidade diversas Issues, neste mento `677`, e tem comunidades com muito mais, veja por exemplo as issues do desenvolvimento do GitLab ou mesmo do Angular (https://github.com/angular/angular/issues neste momento `1854`) entre outros.. 

Como é possível gerenciar tudo isso? Detectar Issues repetidas ou que se complementam? Como rastrear isso? É possível gerenciar isso "apenas" com o  github vanila? 

Gostaria de ver um pouco sobre a visão de vocês sobre isso :smile:

**Comment by klarkc** [comment]
Ah sim, entendi. Cara no caso do angular eu sei q eles usam todo um esquema pra facilitar o gerenciamento, e até a geração de changelogs, coisas como vincular os commits aos PR's (veja o [CONTRIBUTING.md](https://github.com/angular/angular/blob/master/CONTRIBUTING.md) deles, tem tbm [isso aqui](https://github.com/BeeTech-global/bee-stylish/blob/master/commits/README.md) baseado nesse guia). Mas quanto a issues duplicadas e etc, não tem muito oq fazer, a não ser sugerir ao usuário que antes de postar procure se a issue já não existe (o que é meio óbvio na real, mas tem gente que não faz rs).

Hoje o github também oferece o kanban na guia projetos, que é bem útil mas um pouco manual ainda. Fora isso existem outras ferramentas que oferecem integração com o github como o Waffle.io, o ZenHub que funciona dentro do próprio github (extensão do navegador). Essas ferramentas também oferecem um kanban e outras coisas voltadas ao processo, planejamento e gerenciamento de tarefas.
